rero_ils/modules/items/serializers/collector_ext.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2022 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Item collector."""
+
+from rero_ils.modules.documents.api import DocumentsSearch
+from rero_ils.modules.documents.utils import title_format_text_head
+from rero_ils.modules.documents.views import work_access_point
+
+from .collector import Collector
+
+
+class CollectorExt(Collector):
+    """collect data for csv."""
```

Comment:
Could be interesting to add more docstring to explain the usage and purpose of the class. What's the difference with classic Item CSV export